### ISSUE TITLE
Bug: Cannot use HackMD

### ISSUE THREAD

**Issue by haruhikonyan** [weseek/growi#4667]
Environment
------------

### Host

| item     | version |
| ---      | --- |
|OS        ||
|GROWI     |4.4.11|
|node.js   |14.18.1|
|npm       |6.14.15|
|Using Docker|yes(GROWI.cloud)|
|Using [growi-docker-compose][growi-docker-compose]|no|

[growi-docker-compose]: https://github.com/weseek/growi-docker-compose

*(Accessing https://{GROWI_HOST}/admin helps you to fill in above versions)*

### Client

| item     | version |
| ---      | --- |
|OS        |Windows|
|browser   |chrome 95.0.4638.69|

How to reproduce? (再現手順)
---------------------------

1. accsess to edit HackMD(#hackmd)
1. Click to [Start to edit with HackMD]
1. Wait 15 seconds
What happens? (症状)
---------------------
![image](https://user-images.githubusercontent.com/16318813/141724105-daabc0f1-9873-4854-9345-ae11bdf61237.png)

```
TypeError: Cannot read property 'passport' of undefined
    at SessionStrategy.authenticate (/opt/growi/node_modules/passport/lib/strategies/session.js:51:18)
    at attempt (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:360:16)
    at authenticate (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:361:7)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at initialize (/opt/growi/node_modules/passport/lib/middleware/initialize.js:66:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at /opt/growi/packages/app/dist/server/crowi/express-init.js:97:24
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at cookieParser (/opt/growi/node_modules/cookie-parser/index.js:71:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
TypeError: Cannot read property 'passport' of undefined
    at SessionStrategy.authenticate (/opt/growi/node_modules/passport/lib/strategies/session.js:51:18)
    at attempt (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:360:16)
    at authenticate (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:361:7)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at initialize (/opt/growi/node_modules/passport/lib/middleware/initialize.js:66:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at /opt/growi/packages/app/dist/server/crowi/express-init.js:97:24
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at cookieParser (/opt/growi/node_modules/cookie-parser/index.js:71:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
TypeError: Cannot read property 'passport' of undefined
    at SessionStrategy.authenticate (/opt/growi/node_modules/passport/lib/strategies/session.js:51:18)
    at attempt (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:360:16)
    at authenticate (/opt/growi/node_modules/passport/lib/middleware/authenticate.js:361:7)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at initialize (/opt/growi/node_modules/passport/lib/middleware/initialize.js:66:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at /opt/growi/packages/app/dist/server/crowi/express-init.js:97:24
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/opt/growi/node_modules/express/lib/router/index.js:317:13)
    at /opt/growi/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/opt/growi/node_modules/express/lib/router/index.js:335:12)
    at next (/opt/growi/node_modules/express/lib/router/index.js:275:10)
    at cookieParser (/opt/growi/node_modules/cookie-parser/index.js:71:5)
    at Layer.handle [as handle_request] (/opt/growi/node_modules/express/lib/router/layer.js:95:5)
```

What is the expected result? (期待される動作)
-------------------------------------------

- Can edit HackMD
- 

Note
----

- 
-

**Comment by yuki-takei** [comment]
## Note
Timeout duration is set by `timeout: 15000` in `HackmdEditor.jsx `

**Comment by yuki-takei** [comment]
fixed by #4669
